Content.Server/Chat/ChatSystem.cs

```diff
@@ -89,6 +92,8 @@ private void OnLoocEnabledChanged(bool val)
 
         bool shouldCapitalize = (desiredType != InGameICChatType.Emote);
 
+        int channel;
+        (message, channel) = RadioPrefix(source, message);
         message = SanitizeInGameICMessage(source, message, out var emoteStr, shouldCapitalize);
```

Comment:
Is there a reason to get the radio prefix before checking if the entity can speak?